src/persistence/protected_area.js

```diff
@@ -70,6 +70,20 @@ module.exports = (db, { globalBinaryProtectedAreas }) => {
         .groupBy('gbpa.label', 'gbpa.binary_protected');
     },
 
+    /**
+     * Find the the current value of human footprint in the given protected area category
+     * @param {String} categoryName protected area category
+     *
+     * @returns {Object} Object of current human footprint value.
+     */
+    findCurrentHFValue: async (categoryName, year = 2018) => (
+      db('geo_hf as ghf')
+        .innerJoin('global_binary_protected_areas as gbpa', 'ghf.binary_protected', 'gbpa.binary_protected')
```

Comment:
It can't be a simple join for the combined areas

Reply:
done